sphinx_rtd_theme/versions.html

```diff
@@ -7,6 +7,7 @@
       <span class="fa fa-caret-down"></span>
     </span>
     <div class="rst-other-versions">
+      {% if theme_versions_menu %}
```

Comment:
I wonder if this should go above, on the `include versions.html` part? Do we really want all the above HTML in the theme around versions when there isn't a version menu to display?